packages/domain/projet/src/lauréat/statutLauréat.valueType.ts

```diff
@@ -0,0 +1,59 @@
+import { InvalidOperationError, PlainType, ReadonlyValueType } from '@potentiel-domain/core';
+
+const statuts = ['abandonné', 'actif', 'achevé'] as const;
+export type RawType = (typeof statuts)[number];
+
+export type ValueType<Type extends RawType = RawType> = ReadonlyValueType<{
+  statut: Type;
+  estAbandonné: () => boolean;
+  estActif: () => boolean;
```

Comment:
On peut peut-être utiliser un typage plus fort ici pour avoir un lien avec le statut : 


```suggestion
  estAbandonné: () => this is ValueType<'abandonné'>;
  estActif: () => this is ValueType<'actif'>;
  estAchevé: () => this is ValueType<'achevé'>;
```

C'est ce qu'on a pour GF par exemple, mais j'ai pas regardé ce qu'on a ailleurs.